zilliqa/src/cfg.rs

```diff
@@ -135,6 +135,9 @@ pub struct ConsensusConfig {
     /// Minimum remaining time allowing to wait for empty block proposal
     #[serde(default = "empty_block_timeout_default")]
     pub minimum_time_left_for_empty_block: Duration,
+    /// Maximum allowed difference between block creation time and receive time
+    #[serde(default = "maximum_delay_for_received_block_default")]
```

Comment:
I'm not sure there is any way to set this parameter, and so I'm not sure it should exist .. could we decide on either one or the other? It's very much like the timeout parameters we have in ZQ1, which constantly cause spurious view changes ..

Also, I think we should document why we care about validating this and what "creation" and "receive" time mean (and how they are measured!) - eg. it is quite possible for validators to have clocks > 30s out of sync with each other, and if I am joining the network I will definitely receive blocks created more than 1.5s ago! (at least, I would hope the genesis block was created more than 1s ago) - presumably these are excluded from the limit, but how?